cdk/api.py

```diff
@@ -8,15 +8,16 @@ class ZipApi(core.Construct):
 
     def __init__(self, scope: core.Construct, id: str,
             webhook_function,
-            on_demand_function):
+            on_demand_function,
+            lambda_release_alias):
```

Comment:
this env var was missing from stack props, and also not passed in here

Reply:
I started trying to do away with this setting and just hardcoding "live" everywhere. It's in some of the tasks and the codebuild stuff too, so I guess I still need to choose which direction to go on this. I feel like we've never figured out how to take advantage of having this as a variable. Do you want to make an argument for it?